### PR TITLE
[CUBRIDQA-853]add shell_heavy guide.

### PR DESCRIPTION
**Pull request by jinmeihua**
http://jira.cubrid.org/browse/CUBRIDQA-853

To-do: Need to update the URL of [8. shell case standards] and [5. Test Case Specification] after shell and shell_long guides are merged.

### CODE REVIEW

File: doc/shell_heavy_guide.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+
+# 4. Shell_heavy Test Case
+The current isolation test case is located in the https://github.com/CUBRID/cubrid-testcases-private-ex/tree/develop/shell_heavy repository. The directory structure, naming rule, and convention rules of shell_heavy test case are definitely same as that of shell and shell_long test cases. Please refer to [8. shell case standards](https://github.com/CUBRID/cubrid-testtools/blob/a066d8861858fa49885c77796515890b3f35b062/CTP/doc/shell/shell_guide.md#8-shell-case-standards) or [5. Test Case Specification](https://github.com/CUBRID/cubrid-testtools/blob/fbc476b6b76e8a93fe7af21ce726bb29c14380b5/doc/shell_long/README.md#5test-case-specification) to write test case.
```
Comment (fanzaiqiang) [inline doc/shell_heavy_guide.md]:
```suggestion
The current shell_heavy test case is located in the https://github.com/CUBRID/cubrid-testcases-private-ex/tree/develop/shell_heavy repository. The directory structure, naming rule, and convention rules of shell_heavy test case are definitely same as that of shell and shell_long test cases. Please refer to [8. shell case standards](https://github.com/CUBRID/cubrid-testtools/blob/a066d8861858fa49885c77796515890b3f35b062/CTP/doc/shell/shell_guide.md#8-shell-case-standards) or [5. Test Case Specification](https://github.com/CUBRID/cubrid-testtools/blob/fbc476b6b76e8a93fe7af21ce726bb29c14380b5/doc/shell_long/README.md#5test-case-specification) to write test case.
```

File: doc/shell_heavy_guide.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+    _05_addition                _27_aprium_qa     bug_bts_14894    bug_bts_20481  bug_bts_7233  cbrd_21020   cbrd_21361
+    _05_multitbl_query_01       _29_features_920  bug_bts_14980    bug_bts_5188   bug_bts_7313  cbrd_21036   cbrd_21362
+    _05_multitbl_query_02       bug_bts_10295     bug_bts_17460    bug_bts_5349   bug_bts_8675  cbrd_21049   cbrd_21378
+    ```
+4. Install CUBRID
+    ```
+    run_cubrid_install http://192.168.1.91:8080/REPO_ROOT/store_01/10.2.0.8368-b85a234/drop/CUBRID-10.2.0.8368-b85a234-Linux.x86_64.sh
+    ```
+# 3. Regression Tests
+We perform shell_heavy test for every CI builds and perform shell_heavy code coverage test for monthly. You can find the settings from http://10.113.153.154:10086/qaresult/job/job.conf
```
Comment (fanzaiqiang) [inline doc/shell_heavy_guide.md]:
```suggestion
We perform shell_heavy test for every CI builds and perform shell_heavy code coverage test for monthly. You can find the settings from http://qahome.cubrid.org/qaresult/job/job.conf   
```

File: doc/shell_heavy_guide.md [hunk 3]
```diff
@@ -0,0 +1,244 @@
+# 1. Test Objective
+Shel_heavy test is a part of the shell test. The test cases that require big disk size or big memory or with heavy workload are categorized as shell_heavy.
+# 2. Deploy Regression Test Environment
```
Comment (fanzaiqiang) [inline doc/shell_heavy_guide.md]: Hope that CUBRID developers can get guide to execute shell_heavy locally instead of regression deployment.

Comment (jinmeihua) [reply doc/shell_heavy_guide.md]: I've added Execute shell_heavy Test.

**Comment by jinmeihua** [comment]
@fanzaiqiang 
I've modified the guide as your comment. 
Please review again, thank you!

File: doc/shell_heavy_guide.md [hunk 4]
```diff
@@ -0,0 +1,463 @@
+# 1. Test Objective
+`shel_heavy` test is a kind of the shell test. The test cases that require big disk size or big memory or with heavy workload are categorized as shell_heavy.
```
Comment (fanzaiqiang) [inline doc/shell_heavy_guide.md]:
```suggestion
`shel_heavy` test is a kind of the shell test. The test cases that require large disk space or large memory or with heavy workload are categorized as shell_heavy.
```

File: doc/shell_heavy_guide.md [hunk 5]
```diff
@@ -0,0 +1,463 @@
...
+_01_cursor_holdability      _06_issues        bug_bts_13188    bug_bts_17504  bug_bts_5359  bug_xdbms48  cbrd_21070  config
+_02_group_and_async_commit  _09_64bit         bug_bts_14738_1  bug_bts_17550  bug_bts_5868  cbrd_20591   cbrd_21119  issue_5765_timezone_support
+_04_misc                    _26_apricot_qa    bug_bts_14738_2  bug_bts_17599  bug_bts_6995  cbrd_21003   cbrd_21186
+_05_addition                _27_aprium_qa     bug_bts_14894    bug_bts_20481  bug_bts_7233  cbrd_21020   cbrd_21361
+_05_multitbl_query_01       _29_features_920  bug_bts_14980    bug_bts_5188   bug_bts_7313  cbrd_21036   cbrd_21362
+_05_multitbl_query_02       bug_bts_10295     bug_bts_17460    bug_bts_5349   bug_bts_8675  cbrd_21049   cbrd_21378
+```
+## 2.3 Quick Start
+1. Install CUBRID   
+    ```
```
Comment (fanzaiqiang) [inline doc/shell_heavy_guide.md]: Add:
"You may install CUBRID as your way or install via a script file `run_cubrid_install` in CTP".

File: doc/shell_heavy_guide.md [hunk 6]
```diff
@@ -0,0 +1,463 @@
...
+    Total Skip Case:0
+
+    TEST COMPLETE
+    [SHELL] TEST END (Thu Aug 15 20:45:09 KST 2019)
+    [SHELL] ELAPSE TIME: 3279 seconds
+    [may@func43 ~]$ 
+    ```
+## 2.5 Execute one test case
+If you only want to execute one test case, you can simply execute the test case script.   
+In this case, you need to export an init_path at first.
```
Comment (fanzaiqiang) [inline doc/shell_heavy_guide.md]:
```suggestion
In this case, you need to export an `init_path` at first.   
```

File: doc/shell_heavy_guide.md [hunk 7]
```diff
@@ -0,0 +1,463 @@
...
+3 | Test node | shell_heavy | 192.168.1.101 | func26 | CTP, cubrid-testcases-private-ex
+4 | Test node | shell_heavy | 192.168.1.102 | func27 | CTP, cubrid-testcases-private-ex
+5 | Test node | shell_heavy | 192.168.1.103 | func28 | CTP, cubrid-testcases-private-ex
+
+**Controller node** listens to test messages and starts a test when there is a test message. It will distribute test cases to each test node for execution.  
+**Test nodes** execute test cases in parallel.
+
+## 3.2 On Controller Node
+1. Install CTP   
+Follow the steps in [2.1 Install CTP](#21-install-ctp)
```
[the remark this answers is not in the document]
Comment (jinmeihua) [reply doc/shell_heavy_guide.md]: I've modified it.

File: doc/shell_heavy_guide.md (hunk 5 above)
Comment (jinmeihua) [reply doc/shell_heavy_guide.md]: OK, I've added it.

File: doc/shell_heavy_guide.md [hunk 8]
```diff
@@ -0,0 +1,463 @@
...
+    In the current daily qa test, the `shell_heavy`, `shell_long`, and `RQG` tests are using the same test machines, so we use one controllar to listening to the three test messages. The `start_test.sh` is like below in the daily qa test environment. 
+    ```
+    nohup start_consumer.sh -q QUEUE_CUBRID_QA_SHELL_HEAVY_LINUX,QUEUE_CUBRID_QA_RQG,QUEUE_CUBRID_QA_shell_heavy_LINUX -exec run_shell,run_shell,run_shell  &
+    ```
+    Start Consumer   
+    ```
+    $ cd ~
+    $ sh start_test.sh
+    ```
+    Note: If you want to stop listening, execute `sh ~/cubrid_scheduler/stop_consumer.sh`.
```
[the remark this answers is not in the document]
Comment (jinmeihua) [reply doc/shell_heavy_guide.md]: I've fix it.

**Comment by jinmeihua** [comment]
@fanzaiqiang 
I've fixed the guide as your comment. 
Please review, thank you.

File: doc/shell_heavy_guide.md [hunk 9]
```diff
@@ -0,0 +1,423 @@
...
+    Start Consumer   
+    ```
+    $ cd ~
+    $ sh start_test.sh
+    ```
+    Note: If you want to stop listening, execute `stop_consumer.sh`.
+
+## 3.3 On Test Nodes
+1. Install CTP   
+Please refer to the guide to [install CTP in Linux platform](https://github.com/CUBRID/cubrid-testtools/blob/develop/doc/ctp_install_guide.md#1-install-ctp-in-linux-platform) on each test node.
```
Comment (fanzaiqiang) [inline doc/shell_heavy_guide.md]: Update link with https://github.com/CUBRID/cubrid-testtools/blob/develop/doc/ctp_install_guide.md#3-install-ctp-as-regression-test-platform

**Comment by jinmeihua** [comment]
Wait for shell_long guide(https://github.com/CUBRID/cubrid-testtools/pull/558) and shell guide(https://github.com/CUBRID/cubrid-testtools/pull/557) to complete. Then I can update the URL of [8. shell case standards] and [5. Test Case Specification], and merge it.

**Pull request merged**